README.md

```diff
@@ -135,6 +136,8 @@ jobs:
       # optional: files which should be updated for prod
       gitops-prod: |-
         your files
+      # optional: organization of the gitops repository, default: github.repository_owner
+      gitops-organization: <your-organization>
```

Comment:
Out of interest: What is a use case where this should differ? 🤔

Reply:
The `actions/create-github-app-token` action needs an `owner` to set the correct scope for the token. Otherwise, the token is only granted for the current repository. But you want to push to another repository (inside of your organization). Therefore, you need to be able to configure them. Otherwise, people outside of Staffbase can't use that workflow.